public/javascripts/Gallery/src/displays/SeverityDisplay.js

```diff
@@ -7,11 +7,17 @@
  * @param {Boolean} isModal a toggle to determine if this SeverityDisplay is in a modal, or in a card
  * @returns {SeverityDisplay} the generated object
  */
-function SeverityDisplay(container, severity, isModal=false) {
+function SeverityDisplay(container, severity, labelType, isModal=false) {
     let self = this;
     self.severity = severity;
     self.severityContainer = container;
 
+    // list of label types where severity ratings are not supported
```

Comment:
Can you make the small edits to the comments to fit the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)?